source/developers-guide/shopware-5-cli-commands/index.md

```diff
@@ -109,3 +109,54 @@ EOF
 ```
 
 As you can see, a Shopware CLI command is very similar to a Symfony 3 command. You can use any of the features provided by the Symfony Console component, like you would in Symfony 3 itself.
+
+## Add completion for commands
+
+Shopware uses [stecman/symfony-console-completion](https://github.com/stecman/symfony-console-completion) to add completion features you might already know from other CLI.
+This composer package automatically adds completion to any command about the option names.
```

Comment:
```suggestion
This composer package automatically adds completion for options to any command.
```
maybe like this?

Reply:
Yes, it is easier to understand.
I recognized I also need to mention command names itself. I mean `sw:pl` expands to `sw:plugin:` so this is also part of the package that should get mentioned.

Reply:
just let me know, when I could review again :blush:

Reply:
Right here, right now :musical_note: